doc/onboarding.md

```diff
@@ -32,33 +33,38 @@ onboarding session.
 ## Local setup
 
   * git:
-    * make sure you have whitespace=fix: `git config --global --add apply.whitespace fix`
-    * usually PR from your own github fork
+    * Make sure you have whitespace=fix: `git config --global --add apply.whitespace fix`
+    * Always continue to PR from your own github fork
```

Comment:
+1 to this, using branches in Node is not usual behaviour, we should start with the standard workflow.

Maybe:

```diff
-    * Always continue to PR from your own github fork
+    * Continue to raise PRs from your own github fork
```

though